data-canary/scripts/talkactions/player/buy_house.lua

```diff
@@ -5,6 +5,12 @@ function buyHouse.onSay(player, words, param)
 	if housePrice == -1 then
 		return true
 	end
+	
+	local houseBuyLevel = configManager.getNumber(configKeys.HOUSE_BUY_LEVEL)
+	if player:getLevel() < houseBuyLevel then
+		player:sendCancelMessage("You need to be level " ..houseBuyLevel.. " to buy a house.")
+		return false
+	end
```

Comment:
```suggestion
```

It's duplicated.

[Link](https://github.com/opentibiabr/canary/blob/08238198bfb6c6a7e8760c3dda678a84d993465d/data-canary/scripts/talkactions/player/buy_house.lua#L20)